Validator/ViolationFormatterInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Validator;
+
+use FOS\RestBundle\Controller\Annotations\Param;
+use Symfony\Component\Validator\ConstraintViolationInterface;
+use Symfony\Component\Validator\ConstraintViolationListInterface;
+use FOS\RestBundle\Util\ViolationFormatterInterface as LegacyViolationFormatterInterface;
+
+interface ViolationFormatterInterface extends LegacyViolationFormatterInterface
+{
+    /**
+     * Format a Param's ConstraintViolation into a string message.
+     *
+     * @param Param                        $param
+     * @param ConstraintViolationInterface $violation
+     *
+     * @return string
+     */
+    public function format(Param $param, ConstraintViolationInterface $violation);
```

Comment:
you should not redefine the methods of the parent interface. This causes compatibility issues with some PHP versions. Just inherit them from the parent